### ISSUE TITLE
Document or improve the column header decorator. 

### ISSUE THREAD

**Issue by ftrotter** [CareSet/Zermelo#69]
The reporting engine currently converts a column into an automatic "display prettifier".

Usiung this:

```
SELECT 
    abadColumnName AS a_good_column_name,
    thing_count
FROM your_db.your_table.
```

Will result in two columns named "A Good Column Name" and "Thing Count" 

converting underscores to spaces and capitalizing every word is pretty effective as way to improve the general layout of a report. 

But it would be nice if this automatically supported Camel Case as well. So 

```
SELECT
    GoodEnoughColumnName 

```
Should result in "Good Enough Column Name".

We have not currently implemented this because of the complexities of handling variables like HowManyIBMLaptops which is pretty hard to get right using a simple algorithm. 

Perhaps the right way to do this is just to very clearly document how the decorator currently works. Which will force report designers to do:

```
SELECT 
  HowManyIBMLaptops AS How_Many_IBM_Laptops 
```

Just to make things work they way that the want it to. 

-FT

**Comment by tmccormi** [comment]
I agree that there needs to be limited automation, some but not every option... people can override headers if their column names can't be converted.
